app/http.php

```diff
@@ -761,7 +761,7 @@ function (string $runtimeId, ?string $payload, string $path, string $method, arr
                         'runtimeEntrypoint' => $runtimeEntrypoint
                     ]);
 
-                    \curl_setopt($ch, CURLOPT_URL, "http://localhost/v1/runtimes");
+                    \curl_setopt($ch, CURLOPT_URL, "http://127.0.01/v1/runtimes");
```

Comment:
```suggestion
                    \curl_setopt($ch, CURLOPT_URL, "http://127.0.0.1/v1/runtimes");
```

Reply:
Because you can shorten IPv4 like ipv6.

So you can also reach cloudflare 1.0.0.1 DNS address through 1.1  or 127.0.0.1 by 127.1 
Example: [http://1.1](http://1.1)

In general, zeros can be omitted in the middle of IPv4 addresses as long as they are between two numbers. This results in rather strange looking v4 addresses, but is in fact possible.